files/prepare_station.sh

```diff
@@ -105,7 +105,10 @@ get_cluster_info(){
     if sshpass -p "$PASSWORD" scp $STUDENT@$HOSTNAME:./cluster.info cluster.ocp3
     then
         printf "Grabbing cluster info from OCP3 cluster\n"
-        cp cluster.info cluster.orig
+        if [ -f cluster.orig ]; then
+	  cp cluster.orig cluster.info
+	fi 
+    	cp cluster.info cluster.orig
```

Comment:
Why are there two cp?

Reply:
makes it idempotent on multiple logins

Reply:
If I read this right..

if cluster.org exists, then copy it to cluster.info
then copy cluster.info to cluster.orig

This means cluster.orig takes priority if it exists.. and cluster.info data prior to line 108 is discarded.

Reply:
cluster.orig does not exist the very first time, it only has one section for OCP4